src/universal/inject-bank-to-dex.c

```diff
@@ -53,12 +53,23 @@ int main(int argc, char **argv)
     if (choice)
     {
         sav_box_decrypt();
-        int pkmSizeSav = pkx_box_size(genSav), maxSlots = sav_get_max(MAX_SLOTS) - 1, species, isEgg = 0;
+        int pkmSizeSav = pkx_box_size(genSav), slot = 0, maxSlots = sav_get_max(MAX_SLOTS) - 1, species, isEgg = 0, slotsPerBox = (maxSlots + 1) / sav_get_max(MAX_BOXES);
         char *pkxSav = malloc(pkmSizeSav), *pkxInject = malloc(pkmSizeSav), *pkxBank;
         enum Generation genBank;
+        if (genSav == GEN_LGPE) slotsPerBox = 30;
         sav_get_pkx(pkxSav, 0, 0); // save what's already in Box 1 slot 1
 
-        sav_get_pkx(pkxSav, 0, 0);
+        // v9.2.0-656eb68 (stable) compatibility: find a slot with a valid pkm to use for injection
```

Comment:
I only wrote this block of code for the Discord version of this script to allow current PKSM users (running v9.2.0) to run it. It is not necessary otherwise.